.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -3,11 +3,11 @@ name: Feature request
 about: JanusGraph can't do all the things, but maybe it can do your things.
 
 ---
-
+<!--
 For discussion on internal implementation details of JanusGraph or proposing a [major new feature or subproject](https://docs.janusgraph.org/development/#development-decisions) use the [janusgraph-dev](https://groups.google.com/forum/#!forum/janusgraph-dev).
-
+-->
 **Describe the feature:**
-TODO
+<!-- Please add an describe for your feature -->
 
 **Describe a specific use case for the feature:**
-TODO
+<!-- If possible specific use cases for your feature. -->
```

Comment:
_If possible specific use case_ -> _If possible add a description of the specific use case_